docker-compose.yml

```diff
@@ -25,4 +25,14 @@ services:
     build: ./
     image: "${DOCKER_IMAGE}-test:${TAG:-latest}"
     depends_on:
-      - standalone-chrome
\ No newline at end of file
+      - standalone-chrome
+
+  standalone-chrome-travis:
+    image: selenium/standalone-chrome:3.14.0-helium
+    ports:
+      - 4444:4444
```

Comment:
Could we have updated the standalone-chrome service to use `4444:4444` to remove the need to add a new service?

Reply:
if we did it would mess with the local docker environment and bind standalone chrome to the local 4444 port... which would be annoying because those services aren't easily spun down when switching between dev envs